lavamoat/browserify/policy.json

```diff
@@ -4179,6 +4287,16 @@
         "uuid": true
       }
     },
+    "rtcpeerconnection-shim": {
```

Comment:
why are we introducing a webrtc dependencies?

Reply:
we reccomend making a `patch-package` patch that removes the webrtc import from the qr code depenendecy

Reply:
As the above comments, this is introduced by `react-qr-reader`, if we decide to continue using this, we will fork the code and remove the webrtc stuff.

Reply:
removed the `react-qr-reader` to fix this